components/x-gift-article/storybook/with-gift-credits.js

```diff
@@ -38,4 +39,5 @@ exports.fetchMock = (fetchMock) => {
 			redemptionUrl: articleUrlRedeemed,
 			remainingAllowance: 1
 		})
+		.get(`https://enterprise-sharing-api.ft.com/v1/users/me/allowance`, 404)
```

Comment:
**Question:** What about this kind of endpoint? Do you have a way to use environment variables?

Reply:
Specifically for this line, it's just mocking the enterprise sharing URL, other fetches mocked in this test only include the endpoint:
```
.get('/article/gift-credits',
```
and
```
.get(`/article/shorten-url/${encodeURIComponent(articleUrlRedeemed)}`, {
```
but it only works because the api is [initialised](https://github.com/Financial-Times/x-dash/blob/main/components/x-gift-article/src/GiftArticle.jsx#L20-L23) with domain empty. I decided to mock the full enterprise sharing URL so it is clear the difference between the gift api and the enterprise-sharing one. Otherwise someone that look at this code later would think that both services are provided by the same API